src/main/java/com/scottlogic/sort/SortContentLength.java

```diff
@@ -0,0 +1,26 @@
+package com.scottlogic.sort;
+
+import com.scottlogic.SortOrder;
+import com.scottlogic.SortStrategy;
+import com.scottlogic.UserPost;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class SortContentLength implements SortStrategy {
+    @Override
+    public List<UserPost> sort(List<UserPost> inputList, SortOrder sortOrder) {
+        if (inputList == null) {
+            return Arrays.asList();
+        }
+        if (sortOrder.equals(SortOrder.ASC)) {
+            Collections.sort(inputList, Comparator.comparingInt(post -> post.getContents().length()));
```

Comment:
You use Collections.sort a lot, usually I would ask you to do static import but then the name would conflict with the sort method so fair enough.

You can shorten this line by import static Comparator.comparingInt, etc, but since you only use it once here I don't mind either, up to you. (You can resolve this comment yourself)